crypto/bls_multisig.go

```diff
@@ -186,6 +196,8 @@ func NeutralBLSPublicKey() PublicKey {
 // The order of the keys to be removed in the slice does not matter since the removal
 // is commutative. The slice of keys to be removed can be empty.
 // No check is performed on the input public keys.
+// Error returns:
+//  - invalidInputsError if some provided keys are not BLS12 381 keys
```

Comment:
```suggestion
// Expected error returns:
//  - invalidInputsError if at least one key type is not BLS12-381.
```

Reply:
suggestion included in https://github.com/onflow/flow-go/pull/1830/commits/42a51d24cee91f35b355670231e72f7f9aec52c5